### ISSUE TITLE
Require MQTT Instead of Manual Operation of Host to Multicast Send Requests

### ISSUE THREAD

**Issue by CodexLink** [CodexLink/SmartClassroomSystem#13]
This issue corresponds to the possible improvements to the project. Technically, this wasn't going to be worked on because I worked on this one alone. And I'm not quite focused on coding at this point.

So, why issue this? This was intended to be seen by visitors. It's an indication of "there's a flaw on this project".

So, basically, the system wasn't designed as how standards set particular ways to approach filtered sending data to specific devices. We all know this project insists on going to 1:N where N will be the classroom to be controlled on.

Why go for MQTT? Well, standards and proper implementation. I know that its hard to make it sudden but if I changed it right now to do MQTT, there will be lots of mechanisms, unable to perform due to the fact, it will be far more complex as well but understandable when knew enough on how this system works.

TL;DR: I really don't know what to say at least, but I think its best to have something like this such as MQTT in our system. But instead, I did things manually and ended up doing quite complex.

This issue will be guaranteed to be staled. Since, I'm done on it.
